### DISCUSSION TITLE
Open Source is Better with Docs - Check out Pieces' Docs!

### DISCUSSION THREAD

**Discussion by Sophyia7** [pieces-app/opensource#55] [Announcements]
Open Source projects are always better with well-detailed documentation; lucky for you, we're also a big fan of good old-fashioned documentation. Check out our [docs page](https://docs.pieces.app/) on our website, where you can find:

- Installation Support
- Info on our Apps and Plugins
- Product Features and Tutorials
- Highlights and Benefits
And Much More!

**Comment by codeftsaikat** [comment]
ou're referring to a website with great documentation for an open-source project, possibly something related to asset management or a similar domain. If you want to find open-source projects with well-detailed documentation, here’s how you can navigate such resources:

Installation Support: Look for a README.md or docs/installation.md in the project. This should cover how to install the project, including any prerequisites, dependencies, and setup steps.

Apps and Plugins: Many open-source projects integrate with third-party tools, so there might be a section on apps or plugins to extend the functionality. Check if the project supports integrations like CMSs, UI libraries, or frameworks.

Product Features and Tutorials: A well-documented project typically has a Features section where key functionalities are explained, along with code examples and tutorials to help you get started.

Highlights and Benefits: Look for a section that explains why this open-source project exists, its main goals, and how it benefits users. This might also include use cases, performance improvements, and scalability options.

If you're looking for open-source project documentation to guide you in creating your own system, platforms like GitHub or GitLab will allow you to browse other projects' documentation for inspiration.
